h3py/util.pyx

```diff
@@ -0,0 +1,133 @@
+from libc cimport stdlib
+from cython.view cimport array
+
+from h3py.libh3 cimport H3int, H3str
+cimport h3py.libh3 as h3c
+
+# todo: should we use C API functions instead? (stringToH3 and h3ToString)
+cpdef H3int hex2int(H3str h):
+    return int(h, 16)
+
+cpdef H3str int2hex(H3int x):
+    return hex(x)[2:].strip('L') # todo: this strip is... not ideal
```

Comment:
please note in the comment why the `strip('L')` is here

Reply:
good point. added.